dev/bots/run_command.dart

```diff
@@ -147,3 +170,18 @@ Future<void> runCommand(String executable, List<String> arguments, {
     exit(1);
   }
 }
+
+T identity<T>(T x) => x;
```

Comment:
This is only used in a single place, just use a closure

Reply:
Okay, I'll remove it, but I think having it is more readable than:

`utf8.decode(chunks.expand<int>((List<int> ints) => ints).toList());`

with all of the nested parentheses.